From 749f4f76492194fa2556011f26c5f13adfb33d30 Mon Sep 17 00:00:00 2001
From: "Yue.Shen" <Yue.Shen@windriver.com>
Date: Tue, 15 Mar 2016 15:24:51 +0800
Subject: [PATCH] add libubus version number for qa issue

---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e21a046..e846268 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -32,6 +32,7 @@ INCLUDE_DIRECTORIES(${ubox_include_dir})

 ADD_LIBRARY(ubus SHARED libubus.c libubus-io.c libubus-obj.c libubus-sub.c libubus-req.c libubus-acl.c)
 TARGET_LINK_LIBRARIES(ubus ${ubox_library})
+SET_TARGET_PROPERTIES(ubus PROPERTIES VERSION 0.0.0 SOVERSION 0)

 ADD_EXECUTABLE(ubusd ubusd.c ubusd_id.c ubusd_obj.c ubusd_proto.c ubusd_event.c ubusd_acl.c ubusd_monitor.c)
 TARGET_LINK_LIBRARIES(ubusd ${ubox_library} ${blob_library} ${json})
--
1.9.1

