From 89d57b3d4af484cf03025c8aaa12d69a5ebe8502 Mon Sep 17 00:00:00 2001
From: "Shoutao.Li" <Shoutao.Li@windriver.com>
Date: Tue, 22 Sep 2015 17:59:00 +0800
Subject: [PATCH] Shared iptables dynamic shared library path to fw3.

Signed-off-by: Shoutao.Li <Shoutao.Li@windriver.com>
---
 include/xtables.h    | 1 +
 libxtables/xtables.c | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/xtables.h b/include/xtables.h
index 0217267..48b1450 100644
--- a/include/xtables.h
+++ b/include/xtables.h
@@ -409,6 +409,7 @@ struct xtables_globals
 extern "C" {
 #endif
 
+extern const char *xtables_libdir;
 extern const char *xtables_modprobe_program;
 extern struct xtables_match *xtables_matches;
 extern struct xtables_target *xtables_targets;
diff --git a/libxtables/xtables.c b/libxtables/xtables.c
index 2e1c7dc..e9ebd17 100644
--- a/libxtables/xtables.c
+++ b/libxtables/xtables.c
@@ -171,7 +171,7 @@ static const struct xtables_afinfo afinfo_ipv6 = {
 const struct xtables_afinfo *afinfo;
 
 /* Search path for Xtables .so files */
-static const char *xtables_libdir;
+const char *xtables_libdir;
 
 /* the path to command to load kernel module */
 const char *xtables_modprobe_program;
-- 
1.9.1

