From 177b3c08e8ee1acb6c21752a3f73d4bd7f972fdf Mon Sep 17 00:00:00 2001
From: "Yue.Shen" <Yue.Shen@windriver.com>
Date: Wed, 10 Dec 2014 19:05:23 +0800
Subject: [PATCH] dnsmasq: fixes bug that causes unsynchronized resolve file

---
 src/dnsmasq.c | 7 +++++++
 src/dnsmasq.h | 3 +++
 2 files changed, 10 insertions(+)

diff --git a/src/dnsmasq.c b/src/dnsmasq.c
index 1c96a0e..0eb81b3 100644
--- a/src/dnsmasq.c
+++ b/src/dnsmasq.c
@@ -1299,9 +1299,16 @@ void poll_resolv(int force, int do_reload, time_t now)
     else
       {
 	res->logged = 0;
+#if defined __USE_MISC || defined __USE_XOPEN2K8
+	if (force || (statbuf.st_mtime != res->mtime) || (statbuf.st_mtim.tv_nsec != res->mtime_nsec))
+#else
 	if (force || (statbuf.st_mtime != res->mtime))
+#endif
           {
             res->mtime = statbuf.st_mtime;
+#if defined __USE_MISC || defined __USE_XOPEN2K8
+            res->mtime_nsec = statbuf.st_mtim.tv_nsec;
+#endif
 	    if (difftime(statbuf.st_mtime, last_change) > 0.0)
 	      {
 		last_change = statbuf.st_mtime;
diff --git a/src/dnsmasq.h b/src/dnsmasq.h
index 3032546..f9af291 100644
--- a/src/dnsmasq.h
+++ b/src/dnsmasq.h
@@ -532,6 +532,9 @@ struct resolvc {
   struct resolvc *next;
   int is_default, logged;
   time_t mtime;
+#if defined __USE_MISC || defined __USE_XOPEN2K8
+  long mtime_nsec;
+#endif
   char *name;
 };
 
-- 
1.8.3

