From ecb8b7edfd85c58f7434b995e1c84c741e026df3 Mon Sep 17 00:00:00 2001
From: Renxiu Liang <renxiu.liang@windriver.com>
Date: Tue, 3 Jul 2012 14:00:29 +0800
Subject: [PATCH] always show the package info for "smart update"

---
 smart/commands/update.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/smart/commands/update.py b/smart/commands/update.py
index 47a3943..5b4fe2b 100644
--- a/smart/commands/update.py
+++ b/smart/commands/update.py
@@ -85,7 +85,7 @@ def main(ctrl, opts):
     if not newpackages:
         iface.showStatus(_("Channels have no new packages."))
     else:
-        if len(newpackages) <= 10:
+        if len(newpackages) > 0:
             newpackages.sort()
             info = ":\n"
             for pkg in newpackages:
-- 
1.7.11

